bidi-helper.css

```diff
@@ -0,0 +1,58 @@
+/* Some BiDi-proof icons */
+
+[data-icon="back"]:-moz-dir(ltr):before {
+  content: 'left';
+}
```

Comment:
In gaia, we don't skip line between ltr and rtl part of the same rule (AFAIK)o. What do you prefer here?